frankenstein/notes.md

```diff
@@ -0,0 +1,40 @@
+= Project Frankenstein
```

Comment:
It was my intention for this to be markdown, but a brain fart made me use mediawiki format.  Fixed.